src/worldcereal/job.py

```diff
@@ -218,3 +205,120 @@ def collect_inputs(
         out_format="NetCDF",
         job_options={"driver-memory": "4g", "executor-memoryOverhead": "4g"},
     )
+
+
+def _cropland_map(inputs: DataCube) -> DataCube:
```

Comment:
probably this function and the _croptype_map function can be combined into one inference function to avoid duplication? Just adding an additional "product" parameter might do the trick?
But not essential for now I'd say...